src/worker.cc

```diff
@@ -1284,7 +1287,8 @@ to_response_json(const std::string& json, http_request_info_t& request_info, con
 
 worker_t::result_t
 to_response_xml(const std::string& xml, http_request_info_t& request_info, const Api& request) {
-  worker_t::result_t result{false};
+  (void)(request); // Unused variable
```

Comment:
to remove the warning it would be better to just leave the argument unnamed imho